### PR TITLE
docs: v4.1.0 release notes

### PR DESCRIPTION
**Pull request by toteki**
After this merges, I will manually backport by copying this file (without the 4.2.0 section) into a PR against 4.1 branch.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## [v4.1.0](https://github.com/umee-network/umee/releases/tag/v4.0.1) - 2023-02-15
```
Comment (gsk967) [inline CHANGELOG.md:49]: Can you check the tag once

**Review by gsk967 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (toteki) [inline CHANGELOG.md:49]:
```suggestion
## [v4.1.0](https://github.com/umee-network/umee/releases/tag/v4.1.0) - 2023-02-15
```

Comment (toteki) [reply CHANGELOG.md]: oops - changed it (and the link will start working once we tag for real)

**Review by robert-zaremba — APPROVED** [review]
reorganized a bit release notes. The second paragraph should be in v4.2.0 (I've just moved it). When backporting we can just use mergify and fix the conflict.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -6,10 +6,14 @@
...
 Release Procedure is defined in the [CONTRIBUTING](CONTRIBUTING.md#release-procedure) document.
 
-## v4.1.0
+## v4.2.0
 
 - new option is available in `app.toml`: `iavl-lazy-loading` (in general settings). When setting to `true`, lazy loading of iavl store will be enabled and improve start up time of archive nodes.
 
+## v4.1.0
+
+This release contains several fixes designed to make lending and borrowing more resilient during price outages. Short summary of changes is available in the [Changelog](./CHANGELOG.md#v410---2023-02-15)
```
Comment (toteki) [inline RELEASE_NOTES.md:15]:
```suggestion
This release contains several fixes designed to make lending and borrowing more resilient during price outages. Short summary of changes is available in the [Changelog](./CHANGELOG.md)
```

**Comment by toteki** [comment]
Markdown link test is expected to fail (release not yet tagged)

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -52,13 +37,12 @@ $ wget https://raw.githubusercontent.com/CosmWasm/wasmvm/v1.1.1/internal/api/lib
 ```
 
 - Wait for software upgrade proposal to pass and trigger the chain upgrade.
-- Run latest Price Feeder (v2.0.3) - **updated**
```
Comment (robert-zaremba) [inline RELEASE_NOTES.md:55]: Let's keep a note about the minimum price feeder version which has to be used. People are more looking into the release notes than the readme updates.

**Review by robert-zaremba — APPROVED** [review]

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -52,13 +37,12 @@ $ wget https://raw.githubusercontent.com/CosmWasm/wasmvm/v1.1.1/internal/api/lib
...
 - Swap binaries.
 - Restart the chain.
```
Comment (toteki) [inline RELEASE_NOTES.md:41]:
```suggestion
- Restart the chain.
- Run latest Peggo (v1.4.0)
- Run latest Price Feeder (v2.1.0)
```

File: RELEASE_NOTES.md (hunk 3 above)
Comment (toteki) [reply RELEASE_NOTES.md:55]: Added back near this line (there is also a larger note about price feeder around line 17)

**Pull request merged**